app/controllers/trainees/degrees_controller.rb

```diff
@@ -1,26 +1,38 @@
 module Trainees
   class DegreesController < ApplicationController
-    def new
+    def ask
       @degree = trainee.degrees.build
     end
 
+    def finish
+      @degree = trainee.degrees.build(locale_code_params)
+      if @degree.valid?
```

Comment:
Could we update the error messages for this? It current asked the user to select a UK or Non-UK degree option which is not available on the page and may confuse users. It should specify that they select yes or no

Reply:
Fixed, cheers